notebooks/example/anomaly_code.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<p>Can you space out the sublplots? The titles and x-axis labels are running into each other. You might also want to use seaborn, as it creates slightly nicer graphs.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/HTM-applications/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>